### ISSUE TITLE
`Tauri: Update Dependencies` command error

### ISSUE THREAD

**Issue by Tnze** [tauri-apps/tauri-vscode#229]
Press `F1` and execute `Tauri: Update Dependencies`:

![image](https://github.com/tauri-apps/tauri-vscode/assets/19628575/00be0c28-778b-4e8f-b248-ad36922f1fbe)

**Comment by soorya-u** [comment]
@FabianLars I dont think this option [exists](https://github.com/tauri-apps/tauri-vscode/blob/974b7dc08f2f1bad729d90ad571cc19aeb871455/src/extension.ts#L46) anymore.

**Comment by FabianLars** [comment]
Correct, we removed that from the tauri cli ages ago

**Comment by FabianLars** [comment]
Looks like we removed it from the extension as well so it shouldn't show up as an option anymore
